spec/request/users_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'Get /index' do
+    it 'returns http success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+      expect(response).to have_http_status(200)
+    end
+
+    it 'returns the expected content' do
+      get '/users'
+      expect(response.body).to include('This is User index')
+    end
```

Comment:
- [ ] Great job working on the tests but I think it would be great if you add all the tests required and to make sure if all the actions are working including this `If a correct template was rendered.` so I highly recommend adding it as well in index and show page for users 💯🔥🚀✅

![image](https://github.com/CesarHerr/blogApp/assets/58553711/2dcebee5-f0cf-48a0-ba4a-e68436c7ca16)

Reply:
@Reem-lab  Thank you for your review! I will incorporate it! :ok_hand: